packages/xrpl/src/models/transactions/paymentChannelClaim.ts

```diff
@@ -147,12 +153,12 @@ export function validatePaymentChannelClaim(tx: Record<string, unknown>): void {
     throw new ValidationError('PaymentChannelClaim: Channel must be a string')
   }
 
-  if (tx.Balance !== undefined && typeof tx.Balance !== 'string') {
-    throw new ValidationError('PaymentChannelClaim: Balance must be a string')
+  if (tx.Balance !== undefined && !isAmount(tx.Balance)) {
+    throw new ValidationError('PaymentChannelClaim: invalid Balance')
```

Comment:
nit: a better error message would be `PaymentChannelClaim: Balance must be an Amount`

Reply:
ditto for all the others